src/screens/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable react-native-a11y/has-valid-accessibility-descriptors */
+import "react-native-gesture-handler";
+import React, { useState } from "react";
+import {
+  View,
+  StyleSheet,
+  Image,
+  Platform,
+  Dimensions,
+  TextInput,
+  SafeAreaView,
+  Pressable,
+  ScrollView,
+} from "react-native";
+import PropTypes from "prop-types";
+import { Button } from "react-native-elements";
+import { useDispatch } from "react-redux";
+import Text from "../../components/Text";
+import { emailSignIn } from "../../firebase/email_signin";
+import { Role } from "../../types";
+import { AuthUser } from "../../redux/reducers/authReducer/types";
+import { login } from "../../redux/reducers/authReducer";
+
+const styles = StyleSheet.create({
+  root: {
+    flex: 1,
+    alignItems: "flex-start",
+    padding: "5%",
+    backgroundColor: "white",
+    flexDirection: "column",
+    width: "100%",
+  },
+  title: {
+    fontSize: 34,
+    textAlign: "center",
+    marginVertical: 8,
+    color: "#4A4B57",
+  },
+  buttonTitle: {
+    alignSelf: "center",
+    fontWeight: "bold",
+    color: "white",
+  },
+  squareButtonTitle: {
+    alignSelf: "center",
+    fontSize: 20,
+    fontWeight: "400",
+  },
+  image: {
+    width: 100,
+    height: Platform.isPad ? 200 : Dimensions.get("window").height * 0.1,
```

Comment:
Do `Platform.OS === "ios" && Platform.isPad` to pass the eslint checks

Reply:
Not sure why this works but it does